src/components/views/dialogs/AppDownloadDialog.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { FC } from "react";
+
+import { _t } from "../../../languageHandler";
+import SdkConfig from "../../../SdkConfig";
+import AccessibleButton from "../elements/AccessibleButton";
+import QRCode from "../elements/QRCode";
+import Heading from "../typography/Heading";
+import BaseDialog from "./BaseDialog";
+import { IDialogProps } from "./IDialogProps";
+import { Icon as IOSBadge } from "../../../../res/img/badges/ios.svg";
+import { Icon as GooglePlayBadge } from "../../../../res/img/badges/google-play.svg";
+import { Icon as FDroidBadge } from "../../../../res/img/badges/f-droid.svg";
+
+const urlAppStore = "https://apps.apple.com/app/vector/id1083446067";
+const urlGooglePlay = "https://play.google.com/store/apps/details?id=im.vector.app";
+const urlFDroid = "https://f-droid.org/repository/browse/?fdid=im.vector.app";
```

Comment:
We should use the `mobile_builds` config option rather than hardcoding these URLs. Though, I'm sure it's fine to leave these hardcoded URLs here as a fallback

Reply:
Thanks! I knew we had this somewhere, I just couldn’t find it and both incompatible_browser and mobile_guide hardcode all URLs.